python/doc/theory/probabilistic_modeling/conditional_distributions.rst

```diff
@@ -0,0 +1,457 @@
+.. _conditional_distributions:
+
+Conditional distributions
+-------------------------
+
+The library offers some modelisation capacities on conditional distributions:
+
+- Case 1: Create a joint distribution from its conditioned margins,
+- Case 2: Create a distribution which parameters are aleatory,
```

Comment:
aleatory feels weird, maybe random ?
which -> whose

or "Create a distribution with conditioned parameters"

Reply:
"Create a distribution conditioned by its parameters": I think it does not explicitate that the final result is the deconditioned distribution and no more the conditioned one...

Reply:
I propose: "Create a joint distribution using conditioning"